logger.js

```diff
@@ -5,13 +5,12 @@ const logger = winston.createLogger({
   level: 'info',
   format: winston.format.combine(
     winston.format.timestamp(),
-    winston.format.json()
```

Comment:
You do not have to remove the previous logging, you had to create a new one, to logging an `env`

Reply:
Add back old logger